.eslintrc

```diff
@@ -1,10 +1,11 @@
 {
-  "extends": "rapid7/base",
+  "extends": "rapid7/node",
```

Comment:
Eslint setting updates and fixes are here: https://github.com/rapid7/tokend/pull/80